src/game.js

```diff
@@ -2,21 +2,29 @@ const notes = ['C', 'C#', 'D', 'D#', 'E', 'F', 'F#', 'G', 'G#', 'A', 'A#', 'B'];
 export class Game {
   #score;
   #level;
+  #numberOfHintsAvailable;
   #numberOfQuestions;
   #correctAnswers;
   #userAnswers;
   #questionsToReview;
   #publishNewQuestionEvent;
+  #publishNoHintAvailableEvent;
   #publishGameEndEvent;
 
-  constructor(publishNewQuestionEvent, publishGameEndEvent) {
+  constructor(
+    publishNewQuestionEvent,
+    publishNoHintAvailableEvent,
+    publishGameEndEvent
+  ) {
     this.#score = 0;
     this.#level;
+    this.#numberOfHintsAvailable;
```

Comment:
```suggestion
```
Since we're not setting `this.#numberOfHintsAvailable` to a value, we can just declare it above - no need to do it here. Same with `this.#level`, we missed it before 😅

Reply:
Aha! Nice detail!